packages/manager/modules/nasha/src/order/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,16 @@
+{
+  "nasha_order_title": "Commander un NAS-HA",
+  "nasha_order_diskType_header": "Type de disque",
+  "nasha_order_diskType_description": "Choisissez le type de disque en fonction de vos besoins.",
+  "nasha_order_diskType_hdd_description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eu ante non justo eleifend placerat. Aliquam erat volutpat. Donec pellentesque.",
```

Comment:
**question:** `lorem ipsum` is it correct description.

Reply:
Not even close :smile:
The description is pending